Ginger/Ginger/BusinessFlowWindows/xUcBusinessFlowMap.xaml.cs

```diff
@@ -0,0 +1,104 @@
+#region License
+/*
+Copyright © 2014-2019 European Support Limited
+
+Licensed under the Apache License, Version 2.0 (the "License")
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at 
+
+http://www.apache.org/licenses/LICENSE-2.0 
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS, 
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+See the License for the specific language governing permissions and 
+limitations under the License. 
+*/
+#endregion
+
+using System.Windows;
+using System.Windows.Controls;
+using GingerWPF.UserControlsLib.UCTreeView;
+using Amdocs.Ginger.Repository;
+using amdocs.ginger.GingerCoreNET;
+using GingerCore;
+using Amdocs.Ginger.Common.Enums;
+using System.Collections.Generic;
+using Ginger.SolutionWindows.TreeViewItems;
+using System;
+using Amdocs.Ginger.Common;
+
+namespace Ginger.BusinessFlowWindows
+{
+    /// <summary>
+    /// Interaction logic for xUcBusinessFlowMap.xaml
+    /// </summary>
+    public partial class xUcBusinessFlowMap : UserControl
+    {
+        BusinessFlow mBusinessFlow;
+        public BusinessFlow BusinessFlow
+        {
+            get
+            {
+                return mBusinessFlow;
+            }
+            set
+            {
+                mBusinessFlow = value;
+                xBFTextBox.Text = mBusinessFlow.FilePath.Substring(0, mBusinessFlow.FilePath.LastIndexOf("\\")).Substring(mBusinessFlow.ContainingFolderFullPath.Length) + @"\" + mBusinessFlow.ItemName;
+                xGoToAutomateBtn.Visibility = Visibility.Visible;
+            }
+        }
+
+        public string TargetApplication { get; set; }
+
+        public delegate void ElementChangedEventHandler();
+
+        public event ElementChangedEventHandler ElementChangedPageEvent;
+
+        public void ElementChangedEvent()
+        {
+            if (ElementChangedPageEvent != null)
+            {
+                ElementChangedPageEvent();
+            }
+        }
+
+        public xUcBusinessFlowMap()
+        {
+            InitializeComponent(); 
+        }
+
+        private void xSelectBF_Click(object sender, RoutedEventArgs e)
```

Comment:
see comments i mentioned on POMGeneralDetails page cs on how the selected BF should be set/loaded within this class

Reply:
changed as per POMGeneralDetails